### PR TITLE
Django configurations

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by dougalsutherland** [comment]
You shouldn't delete the `flake8` recipe that happens to be in the repo in your PR, just leave it in place. (This might be why the CircleCI build failed, but the log file won't load for me right now.)

**Comment by sannykr** [comment]
@dougalsutherland : Thanks for your suggestion. I think it's an issue related to license and I need to add the license file manually.

File: recipes/flake8/meta.yaml [hunk 1]
```diff
@@ -1,63 +0,0 @@
-{%set name = "flake8" %}
```
Comment (jakirkham) [inline recipes/flake8/meta.yaml]: Please undelete this.

Comment (sannykr) [inline recipes/flake8/meta.yaml]: @jakirkham : if I undelete that file, there will be two meta.yaml files. Actually, while building flake8, I forgot to create a new branch for flake8, and created recipe at master of the forked repo. When I made a branch for django-configurations, it had a directory flake8 with an yaml file. So, I deleted that file.
If I undelete that file, there will be two yaml files. Will that cause a problem?

Comment (sannykr) [inline recipes/flake8/meta.yaml]: I think I can also create a different branch, and make a pull request from there. Will that be fine?

Comment (dougalsutherland) [inline recipes/flake8/meta.yaml]: @sannykr You should probably rebase your commits to the latest master and force-push to this branch.

Comment (jakirkham) [inline recipes/flake8/meta.yaml]: Yeah just merging with `master` may be enough to fix this.

**Pull request closed without merging**